sysid-application/src/main/native/cpp/view/Logger.cpp

```diff
@@ -106,11 +106,12 @@ void Logger::Display() {
                      m_isRotationalUnits ? ImGuiInputTextFlags_ReadOnly
                                          : ImGuiInputTextFlags_None);
   sysid::CreateTooltip(
-      "The logger assumes that the code will be sending recorded rotations "
+      "The logger assumes that the code will be sending recorded output shaft "
+      "rotations "
       "over NetworkTables. This value will then be multiplied by the units per "
       "rotation to get the measurement in the units you specified.\n\nFor "
       "non-rotational units (e.g. meters), this value is usually the wheel "
-      "diameter times pi.");
+      "diameter times pi (does not include gearing).");
```

Comment:
Maybe should not include gearing is a better phrase since saying does not leaves room for assuming maybe it should.

Reply:
How so?

Reply:
Why bring up gearing at all?

Reply:
Ah that makes sense. The intent was to clarify the gearing thing because we did say the contrary on a Chief Delphi post. But I can see how that can make things worse for other teams.